redash/query_runner/athena.py

```diff
@@ -33,6 +34,11 @@
 }
 
 
+class SimpleFormatter(object):
+    def format(self, operation, parameters=None):
+        return operation
```

Comment:
@laughingman7743 I'm using this to avoid escaping the '%' character in Athena queries. WDYT?